style.css

```diff
@@ -0,0 +1,81 @@
+#header {
+}
+body {
+  background-color: rgb(185, 199, 247);
+}
+
+.website-main {
+  display: flex;
+  width: 100%;
+}
+
+#header-img {
+  position: fixed;
+  width: 60px;
+  height: 60px;
+  float: left;
+  z-index: 1;
+}
+#nav-bar {
+  position: fixed;
+  float: right;
+  width: 100%;
+  font-size: 24px;
+  z-index: 1;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  display: inline;
+  float: right;
+}
+.nav-link {
+  display: block;
+  padding: 8px;
+  background-color: #8369e9;
+}
+
+#video {
+  position: relative;
+  top: 100px;
+  width: 100%;
+}
+
+#form {
+  position: fixed;
+  top: 80px;
+}
+
+#email {
```

Comment:
I would suggest removing any unnecessary selectors that are not being used to improve the styling in that way you maintain a more organized CSS code.